src/RwaToken.sol

```diff
@@ -17,8 +17,9 @@ pragma solidity 0.5.12;
 
 contract RwaToken {
     // --- ERC20 Data ---
-    string  public constant name     = "RWA-002";
-    string  public constant symbol   = "RWA002";
+    string  public name;
+    string  public symbol;
```

Comment:
It would be preferable to move to compiling with some `0.6` version so that these could be marked `immutable`, but reading `name` and `symbol` is rare on-chain, so I guess it's not strictly necessary.